basxconnect/core/layouts/editperson.py

```diff
@@ -436,38 +434,31 @@ def revisionstab(request):
 def relationshipstab(request):
     return (
         _("Relationships"),
-        layout.datatable.DataTable.from_model(
-            Relationship,
-            hg.F(
-                lambda c, e: c["object"].relationships_to.all()
-                | c["object"].relationships_from.all()
+        layout.grid.Grid(
+            layout.form.FormsetField.as_datatable(
+                "relationships_to",
+                [
+                    layout.datatable.DataTableColumn("", hg.C("object")),
```

Comment:
I would have the header "Person A" here. It is confusing to have only a column with header "Person B".

Another way would be to completely remove the column which contains the current person and have that information only in the datatable header, but I actually like having the A and the B column, because it maybe helps the user to understand the actual datamodel.

For the title of the datatable I would find something like following a bit easier to understand: "Relationships from Samuel to _Person B_"

Reply:
Good suggestion, I will make the changes.
Yes, I agree with having two columns is an advantage.
I am also interested what @frederikbugglin, we can discuss it in the next meeting.